libs/blocks/global-navigation/global-navigation.css

```diff
@@ -391,6 +391,16 @@ header.global-navigation {
   border-radius: var(--feds-radius-utilityIcon);
   color: #4B4B4B;
   white-space: nowrap;
+  border: none;
+  text-align: start;
+  font-family: inherit;
+  font-size: inherit;
+  line-height: inherit;
```

Comment:
Did you try with just `font: inherit`? Since it's a shorthand, that should inherit all 3 of these properties + the `font-size`.